charts/microservice/values.yaml

```diff
@@ -1,12 +1,42 @@
-replicaCount: 1
+# -- global variables  
+global:
+  replicaCount: 1
+  fullnameOverride: "nginx"
+  nameOverride: ""
 
-fullnameOverride: "nginx"
-nameOverride: ""
-image:
-  repository: nginx
-  pullPolicy: IfNotPresent
-  tag: "latest"
+# -- Object that configures Deployment instance
+deployment:
+  # -- Override default container image format
+  image:
+    # -- Image repository
+    repository: "nginx"
+    # -- Image tag
+    tag: "latest"
+    # -- Variable used to control when the image should be fetched
+    pullPolicy: always
+  # -- Definition of the liveness probe
+  livenessProbe: 
```

Comment:
Should be optional
Can we have a warning mentioning this should have been present